test/cypress/integration/version/versioning.spec.js

```diff
@@ -1,7 +1,7 @@
 describe('Versioning', function() {
-    it('is visible on the start page', function() {
-      cy.visit('/')
-      var expected = '0.11'
-      cy.get('#version').should('contain', expected)
-    })
-  })
\ No newline at end of file
+  it('is visible on the start page', function() {
+    cy.visit('/')
+    var expected = '0.11'
```

Comment:
Meh, can be inlined 😄 
Then you can read the next line as regular english sentence.

Reply:
No probs:)